ext/zend_test/test.c

```diff
@@ -726,7 +744,10 @@ ZEND_GET_MODULE(zend_test)
 /* The important part here is the ZEND_FASTCALL. */
 PHP_ZEND_TEST_API int ZEND_FASTCALL bug78270(const char *str, size_t str_len)
 {
-	return (int) zend_atol(str, str_len);
+	char * copy = zend_strndup(str, str_len);
```

Comment:
Any reason why this copy is needed? `ZEND_ATOL` (`atol`, `atoll` or `_atoi64`) shouldn't modify the string.

Reply:
`zend_atol` took a length, but `ZEND_ATOL` doesn't, so I wanted to add a 0 byte at the relevant place without modifying `str`